packages/dapp/src/components/Widget/Widget.tsx

```diff
@@ -140,7 +140,6 @@ export function Widget({ starterVariant }) {
           maxPriceImpact: 0.4,
         },
       },
-      buildSwapUrl: true,
```

Comment:
@chybisov This "buildSwapUrl" is not used any more and there is no other property which replaces this? Here, I simply deleted it as it is not part of the WidgetConfig types anymore.

Reply:
@dennyscode it was renamed to `buildUrl` 🙂